index.html

```diff
@@ -65,59 +65,52 @@ <h2 class="section_heading">
         We offer a handpicked range of cabins for nature lovers in Europe to live and work
         from for desired time.
       </h2>
-      <div class="flex-j-center flex">
```

Comment:
does it need  `flex-j-center` if it doesn't have flex?

Reply:
it does,  the flex class have had "flex" property but now I fixed it and decided that flex-j-center also will have it for itself. :)